tests/bson/bson-timestamp-set_state_error-002.phpt

```diff
@@ -10,15 +10,15 @@ echo throws(function() {
 }, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
 
 echo throws(function() {
-    MongoDB\BSON\Timestamp::__set_state(['increment' => -2147483648, 'timestamp' => 5678]);
+    MongoDB\BSON\Timestamp::__set_state(['increment' => -2147483647, 'timestamp' => 5678]);
```

Comment:
In that case, this isn't really testing a boundary condition, is it? I was testing `-2147483648` because it should have been equal to `4294967295` in unsigned form.

I'd be OK with remove these test cases unless you really want to keep them.